### Why
Developers needed a way to catch failed exceptions from calls to Beamable services inside a Microservice. 

### Configuration
none

### How
Inside a Microservice, if you make a request to a Beamable service, and the Beamable service fails, you can now catch the exception
using the `RequesterException` base type. 

```csharp
using Beamable.Common.Api;
using Beamable.Server;
using System.Threading.Tasks;

namespace Beamable.Microservices
{
	[Microservice("Doop1")]
	public class Doop1 : Microservice
	{
		[ClientCallable]
		public async Task<string> ServerCall()
		{
			try
			{
				await Services.Inventory.SetCurrency("does-not-exist-so-it-will-trigger-a-400", 3);
			}
			catch (RequesterException ex)
			{
				return $"{ex.Method.ToReadableString()} / {ex.Payload} / {ex.Prefix} / {ex.Status} / {ex.Uri} / {ex.RequestError.error} / {ex.RequestError.message} / {ex.RequestError.service}";
			}

			return "okay";
		}
	}
}

```

### Prefab
none

### Editor
none

### Notes
The same base type can be used in Unity SDK code. The `RequesterException` is the base type exception. In a Microservice, if you checked the _type_ of the exception, it would be a `WebsocketRequesterException`, and in Unity, it would be a `PlatformRequesterException`. But they both extend for the common `RequesterException` which has many available error details.